inst/extdata/sectoral/prom-reporting-primaryproduction-mapping.csv

```diff
@@ -0,0 +1,11 @@
+﻿Reporting,ENERDATA,IEA
+Primary Energy,NA,TOTAL
+Primary Energy|Biomass,Primary production of biomass.Mtoe,PRIMSBIO
+Primary Energy|Coal,Primary production of Coal and lignite.Mtoe,BITCOAL
```

Comment:
shouldn't the IEA column for Coal include also LIGNITE, ANTCOAL, etc.?

Reply:
ok
![image](https://github.com/user-attachments/assets/aa67a65f-1d2b-45c3-b13f-b0c0bd254894)